anago

```diff
@@ -650,6 +646,9 @@ get_build_candidate () {
     # If the branch is a 3-part branch (ie. release-1.2.3)
     if [[ $RELEASE_BRANCH =~ $BRANCH_REGEX ]] && \
        [[ -n ${BASH_REMATCH[4]} ]]; then
+      ((FLAGS_official)) \
```

Comment:
When I tried it before, the script required me to add `--official` on that first step. So if I understand you correctly, this PR fixes that by moving the check to a different place? Thanks.